crates/sui-adapter/src/adapter.rs

```diff
@@ -509,6 +525,25 @@ pub fn verify_and_link<
     Ok(vm)
 }
 
+/// Given a Vec of `CompiledModule`, runs each module with both the Move VM verifier
+/// and the Sui verifier.
+pub fn run_move_and_sui_bytecode_verifier(
+    modules: &[CompiledModule],
+) -> Result<(), ExecutionError> {
```

Comment:
Any reason to have this in the `adapter`? Feels fine to put it over in the `transaction_input_checker` since that is the only place that cares about this code

Reply:
@tnowacki, I suppose this was to reuse code that is needed in both places? 
It seems to make sense to me, how would you do it otherwise? curious....